.sync.yml

```diff
@@ -18,3 +16,66 @@ Gemfile:
 spec/spec_helper.rb:
   mock_with: ":rspec"
   coverage_report: true
+'.travis.yml':
+  deploy_to_forge:
+    enabled: false
+  branches:
+    - release
+  user: puppet
+  secure: ''
+  includes:
+    - bundler_args:
+      dist: trusty
+      env: PLATFORMS=deb_puppet5
+      rvm: 2.5.3
+      before_script:
+        - bundle exec rake 'litmus:provision_list[travis_deb]'
+        - bundle exec bolt command run 'apt-get install wget -y' --inventoryfile inventory.yaml --nodes='localhost*'
+        - bundle exec rake 'litmus:install_agent[puppet5]'
+        - bundle exec rake litmus:install_module
+      script:
+        - bundle exec rake litmus:acceptance:parallel
+      services: docker
+      stage: acceptance
+      sudo: required
+    - bundler_args:
+      dist: trusty
+      env: PLATFORM=centos:deb_puppet6
+      rvm: 2.5.3
+      before_script:
+        - bundle exec rake 'litmus:provision_list[travis_deb]'
+        - bundle exec bolt command run 'apt-get install wget -y' --inventoryfile inventory.yaml --nodes='localhost*'
+        - bundle exec rake 'litmus:install_agent[puppet6]'
+        - bundle exec rake litmus:install_module
+      script:
+        - bundle exec rake litmus:acceptance:parallel
+      services: docker
+      sudo: required
+      stage: acceptance
+    - bundler_args:
+      dist: trusty
+      env: PLATFORMS=el_puppet5
+      rvm: 2.5.3
+      before_script:
+        - bundle exec rake 'litmus:provision_list[travis_el]'
```

Comment:
here should eb travis_el7, travis_el6
please check in [provision file](https://github.com/puppetlabs/puppetlabs-vcsrepo/blob/7cf595421a376940c2bb0bb29b019b19915a95e5/provision.yaml#L8)

Reply:
Good spot @lionce - thanks